### ISSUE TITLE
Referencing the latest I-D BibXML causes resulting ENTITY to lose link 

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/asciidoctor-rfc#94]
I encountered this in the rfc-asciirfc-minimal document, where I tried to cite the latest AsciiRFC version (I manually fetched the BibXML reference from the xml2rfc.ietf.org site). However, the generated RFC XML will have the ENTITY tag href “empty”. Once I change the included BibXML to refer to a n-1 version (05), the entity href came back. 

Maybe there is an off by one error somewhere. In any case it is very strange behavior. No error messages were given too.

**Comment by ronaldtse** [comment]
I also tried the flush-bibliography option but it doesn’t make any difference.

**Comment by opoudjis** [comment]
I was able to generate successfully, with draft version 6 for the Asciidoc RFC. I got an error on both I-Ds before flushing bibliography, but was then able to generate it successfully after flushing. Did you do the following?

rm ~/.asciidoc-rfc-biblio-cache.json
bundle exec asciidoctor **-a flush-caches=true** -r ./lib/glob-include-processor.rb -r asciidoctor-rfc -b rfc2 -a flush-biblio=true draft-asciirfc-minimal.adoc --trace > draft-asciirfc-minimal.xml
